.github/workflows/php_ci.yml

```diff
@@ -17,10 +11,10 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
     
       - name: Setup PHP
-        uses: shivammathur/setup-php@86e1ccdd8ddc47bffc29bf667143f363a4cdfdbc
```

Comment:
Although we don't do it everywhere like we should, GitHub actually recommends pinning to a SHA for actions; especially third-party actions.  I'm fine with moving to a newer version but we should still pin to a sha here please

Reply:
Fixed.